app/src/main/java/fr/free/nrw/commons/feedback/FeedbackContentCreator.java

```diff
@@ -32,6 +34,9 @@ public void init() {
         /*
          * Construct the feedback section title
          */
+
+        final SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss", Locale.ENGLISH);
```

Comment:
This will record time in the user device's timezone right?
Would you mind taking the **UTC time** instead?
Doing so will avoid surprising situations where times seem to be in mixed order.
Thanks! :-)

Reply:
I have added the date and time in UTC.